frontend/src/App.js

```diff
@@ -93,6 +92,7 @@ function App() {
 
         {postItemComponents}
         <p>{userID}</p>
+        <GithubActivity userID={userID}/>
```

Comment:
Shouldn't this be integrated with the postItemComponents? the way this is right now, it would appear under all the posts whereas I think that it should be within it.

Reply:
Feel free to change where it appears. I wasn't sure about that.